lizrd/core/llm.py

```diff
@@ -80,6 +80,16 @@ def forward(self, x):
         return out + x
 
 
+@ash.check("... -> ... ")
+class Parallel(nn.Module):
+    def __init__(self, *layers):
+        super(Parallel, self).__init__()
+        self.layers = layers
```

Comment:
do _not_ use a normal list of nn.modules as attributes of an nn.module, it does not register properly. Use [ModuleList](https://pytorch.org/docs/stable/generated/torch.nn.ModuleList.html).

Reply:
Good point